### PR TITLE
TECH-562: Add cucumber test for Registration - Applicant - POST /services/{serviceId}/applications

### PR DESCRIPTION
**Pull request by mgrochalskisoldevelo**
Added cucumber tests, mocks and gherkin scenarios for POST /services/:serviceId/applications endpoint

TICKET: https://govstack-global.atlassian.net/browse/TECH-562

### CODE REVIEW

File: test/openAPI/features/applicant_services_serviceid_applications.feature [hunk 1]
```diff
@@ -0,0 +1,43 @@
+@method=POST @endpoint=/services/{serviceId}/applications
+Feature: API endpoint that allows to submit an application file as an applicant
+  
+  Send an application file including all documents and form data to submit a registration request to be processed by operators.
+
+  @smoke
+  Scenario: Successfully created a new application file in the system smoke type test
+
+    Given An applicant wants to submit an application
+    When POST request to submit an application is sent with given I-M-C header and "f7d33db0-2809-484e-a780-76b7ccd4ecbf" as serviceId parameter
```
Comment (KarolinaKopacz) [inline test/openAPI/features/applicant_services_serviceid_applications.feature]: Not all user can be familiar what the I-M-C means
```suggestion
    When POST request to submit an application is sent with given Information-Mediator-Client header and "f7d33db0-2809-484e-a780-76b7ccd4ecbf" as serviceId parameter
```

File: test/openAPI/features/support/helpers/helpers.js [hunk 2]
```diff
@@ -44,6 +45,49 @@ module.exports = {
       },
     ],
   },
+  // services/{serviceId}/applications
```
Comment (KarolinaKopacz) [inline test/openAPI/features/support/helpers/helpers.js]:
```suggestion
  // applicant_services_serviceid_applications
```

File: test/openAPI/features/support/helpers/helpers.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ module.exports = {
     value: 'eGovStack/GOV/90000009/eregistrations-dev',
   },
   defaultExpectedResponseTime: 15000,
+  // data statistics
```
Comment (KarolinaKopacz) [inline test/openAPI/features/support/helpers/helpers.js]:
```suggestion
  // data_statistics
```

File: test/openAPI/features/support/helpers/helpers.js [hunk 4]
```diff
@@ -44,6 +45,49 @@ module.exports = {
...
+          documentUrls: {
+            type: 'array',
+          },
+        },
+        required: ['applicantId'],
+      },
+    },
+    required: ['serviceId', 'applicationData'],
+  },
+  // services
```
Comment (KarolinaKopacz) [inline test/openAPI/features/support/helpers/helpers.js]:
```suggestion
  // services_services
```

File: test/openAPI/features/support/helpers/helpers.js (hunk 3 above)
Comment (KarolinaKopacz) [reply test/openAPI/features/support/helpers/helpers.js]: To be consistent with another helpers.js files use here the test file name

**Review by KarolinaKopacz — CHANGES_REQUESTED** [review]

**Review by ingmarvali — APPROVED** [review]

**Review by KarolinaKopacz — APPROVED** [review]

**Pull request merged**